lib/shortcuts.js

```diff
@@ -6,20 +6,28 @@ const FileSync = require('lowdb/adapters/FileSync');
 const adapter = new FileSync('shortcuts.json');
 const db = low(adapter);
 
-db.defaults({ shortcuts: [] }).write();
+db.defaults({ users: [] }).write();
 
-const addShortcut = (name, emojis) => {
-  if (db.get('shortcuts').find({ name }).value()) {
+const addShortcut = (userId, name, emojis) => {
+  if (!db.get('users').find({ userId }).value()) {
```

Comment:
Is this code better ?
```suggestion
  const user = db.get('users').find({userId});
  if (user.value()) {
    [...]
  }
```

Reply:
I apply it in my last commit